### DISCUSSION TITLE
Host Authelia without publishing port

### DISCUSSION THREAD

**Discussion by dermot-mcg** [authelia/authelia#6515] [Q&A] [answered]
I'm using Authelia and its protecting my subdomains which Nginx Proxy Manager proxies to docker containers. I've notices authelia is the only service that needs to publish a port (9091), which means its accessible via my_ip_address:9091. This means it bypasses SSL. Wondering how I can run it without publishing a port? It seems at the moment the subdomains expect to connect to 9091 for auth

**Comment by james-d-elliott** [comment, marked as answer]
Authelia doesn't need to publish a port, this is a bad assumption. See the official example which has been tested and no ports are published or mapped to the host for authelia, and are only done so for NPM: https://www.authelia.com/integration/proxies/nginx-proxy-manager/

**Reply by dermot-mcg** [reply]
It worked now, my mistake, I think I just needed to recreate the ngxinx proxy host. Thanks
